qemu/tests/cfg/usb.cfg

```diff
@@ -390,6 +390,7 @@
                     # Please configure temp repository url for package usbredir-server
                     # temprepo_url = "http://xxx"
         - usb_smartcard_sharing:
+            no ppc64 ppc64le
```

Comment:
`usb_redir` use `spicevmc` too, i guess it can be disabled for ppc too ?

Reply:
Thanks for the reminder and they should be disabled too.